### PR TITLE
Add changelog for v0.10.0

### PR DESCRIPTION
**Pull request by chrishenzie**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Adds the changelog for the v0.10.0 release of csi-lib-utils.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

/assign @msau42

### CODE REVIEW

File: CHANGELOG/CHANGELOG-0.10.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+## Changes by Kind
+
+### Feature
+
+- Adds mappings for PV access modes to new CSI access modes: `SINGLE_NODE_SINGLE_WRITER` and `SINGLE_NODE_MULTI_WRITER`. ([#89](https://github.com/kubernetes-csi/csi-lib-utils/pull/89), [@chrishenzie](https://github.com/chrishenzie))
+
+### Other (Cleanup or Flake)
+
+- Csi-lib-utils library package dependencies are updated to make use of latest kube version v1.21.0 and also v1.4.0 version of csi spec which include many bug fixes and enhancements, with this release. ([#86](https://github.com/kubernetes-csi/csi-lib-utils/pull/86), [@humblec](https://github.com/humblec))
+- Sidecars exit without output about goroutines when loosing the connection to the driver ([#81](https://github.com/kubernetes-csi/csi-lib-utils/pull/81), [@pohly](https://github.com/pohly))
```
Comment (msau42) [inline CHANGELOG/CHANGELOG-0.10.md:14]: I would prefix this with "Fixes issue where"

Comment (chrishenzie) [reply CHANGELOG/CHANGELOG-0.10.md]: Done, also fixed typo "loosing"

File: CHANGELOG/CHANGELOG-0.10.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+### Feature
+
+- Adds mappings for PV access modes to new CSI access modes: `SINGLE_NODE_SINGLE_WRITER` and `SINGLE_NODE_MULTI_WRITER`. ([#89](https://github.com/kubernetes-csi/csi-lib-utils/pull/89), [@chrishenzie](https://github.com/chrishenzie))
+- Updates Kubernetes dependencies to v1.22.0 ([#98](https://github.com/kubernetes-csi/csi-lib-utils/pull/98), [@chrishenzie](https://github.com/chrishenzie))
+- Updates container-storage-interface dependency to v1.5.0 ([#90](https://github.com/kubernetes-csi/csi-lib-utils/pull/90), [@chrishenzie](https://github.com/chrishenzie))
+
+### Other (Cleanup or Flake)
+
+- Csi-lib-utils library package dependencies are updated to make use of latest kube version v1.21.0 and also v1.4.0 version of csi spec which include many bug fixes and enhancements, with this release. ([#86](https://github.com/kubernetes-csi/csi-lib-utils/pull/86), [@humblec](https://github.com/humblec))
```
Comment (msau42) [inline CHANGELOG/CHANGELOG-0.10.md:15]: Is this relevant anymore since we updated to 1.22 and 1.5?

Comment (chrishenzie) [reply CHANGELOG/CHANGELOG-0.10.md]: No, good point, removed it

**Pull request merged**